docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-mesh-validate.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: mesheryctl system mesh validate | Meshery
+permalink: reference/mesheryctl/mesh/validate
+type: reference
+display-title: "false"
+language: en
+categories: en
+list: exclude
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl system mesh validate
+
+<!-- Description of the command. Preferably a paragraph -->
+## Description
+
+{% for subcommand_hash in site.data.mesheryctlcommands.meshes.validate.subcommand %}{% assign subcommand = subcommand_hash[1] %}
+{{ subcommand.description }}
+{% endfor %}
+
+<!-- Basic usage of the command -->
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  mesheryctl mesh validate [flags]
+  </div>
+</pre>
+
+## Examples
+
+<h6>(required) adapter to use for validation. Defaults to "meshery-osm:10010"<h6>
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  mesheryctl mesh validate --adapter [name of the adapter]
+  </div>
+</pre>
+<h6>Kubernetes namespace to be used for deploying the validation tests and sample workload<h6>
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  mesheryctl mesh validate --namespace [namespace to be used]
+  </div>
+</pre>
+<h6>(required) specification to be used for conformance test. Defaults to "smi"<h6>
+<pre class="codeblock-pre">
+  <div class="codeblock">
+  mesheryctl mesh validate --spec [specification to be used for conformance test]
+  </div>
+</pre>

```

Comment:
Now I realised what you meant by manual 🤣

Reply:
Let's see if @leecalcote accepts this. TBH IDK if this can be maintainable in future

Reply:
@alphaX86 : Yep, actually it's not possible to fixed as others page do, I fixed one with `delete` earlier.
Test approximately all possibles ways!

@leecalcote : I will take the further maintainaince of this page.